src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -364,13 +364,16 @@ public HttpRedirect doGenerateBundleAsync(StaplerRequest2 req, StaplerResponse2
         List<Component> syncComponent =
                 components.stream().filter(c -> !c.canBeGeneratedAsync()).toList();
         if (!syncComponent.isEmpty()) {
-            File outputDir = new File(SUPPORT_BUNDLE_CREATION_FOLDER + "/" + taskId);
-            if (!outputDir.exists()) {
-                if (!outputDir.mkdirs()) {
-                    throw new IOException("Failed to create directory: " + outputDir.getAbsolutePath());
+            Path outputDir = SUPPORT_BUNDLE_CREATION_FOLDER.resolve(taskId.toString());
+            if (!Files.exists(outputDir)) {
+                try {
+                    Files.createDirectories(outputDir);
+                } catch (IOException e) {
+                    throw new IOException("Failed to create directory: " + outputDir.toAbsolutePath(), e);
```

Comment:
Probably unnecessary to wrap and rethrow; the original `IOException` would include sufficient details.